moped-editor/src/views/projects/projectView/ProjectComponents/index.js

```diff
@@ -192,6 +189,13 @@ export default function MapView({
     );
   };
 
+  /* Start creating and clear clicked component and draft edit states to deselect component */
+  const onStartCreatingComponent = () => {
+    createDispatch({ type: "start_create" });
+    editDispatch({ type: "clear_draft_component" });
+    setClickedComponent(null);
+  };
```

Comment:
Part of this used to be buried in the `useCreateComponent` hook. Moving it here hopefully makes it easier to see all of the actions happening when the **New Component** button is clicked in the list on the left of the map.

Reply:
yes—this is a really nice lil improvement!